projects/hslayers/src/components/add-data/catalogue/layer-descriptor.model.ts

```diff
@@ -15,8 +15,9 @@ export interface HsAddDataLayerDescriptor {
   id?;
   workspace?: string;
   editable?: boolean;
-  sld?: {
+  style?: {
     url: string;
+    type: string;
```

Comment:
Can we put a literal type 
```suggestion
    type: 'sld' | 'qml';
```
here?

Reply:
Probably yes. It might break silently this way though if layman would start to support something more.